Source/WebKit/Platform/SharedMemory.h

```diff
@@ -99,11 +100,11 @@ class SharedMemory : public ThreadSafeRefCounted<SharedMemory> {
 #if USE(UNIX_DOMAIN_SOCKETS)
         mutable IPC::Attachment m_attachment;
 #elif OS(DARWIN)
-        mutable mach_port_t m_port { MACH_PORT_NULL };
+        mutable MachSendRight m_handle;
```

Comment:
Call this one m_sendRight as well for consistency?

Reply:
The eventual idea is to call all the other ports' handles m_handle and move the decode/encode code to SharedMemory.cpp to be shared. The textual code will be the same for all.